packages/viewer/src/lib/rendering/matrices.ts

```diff
@@ -155,3 +155,29 @@ export function makeLookAtMatrix(
   ];
   /* eslint-enable prettier/prettier */
 }
+
+export function experimentalMakeTransformationMatrix(
+  normal1: Vector3.Vector3,
+  position1: Vector3.Vector3,
+  normal2: Vector3.Vector3,
+  position2: Vector3.Vector3
+): Matrix4.Matrix4 {
+  const translationDeltaMatrix = Matrix4.makeTranslation(
+    Vector3.subtract(position2, position1)
+  );
+  const angle = Vector3.angleTo(normal2, normal1);
+
+  const axisDirection = Vector3.normalize(Vector3.cross(normal1, normal2));
```

Comment:
Cross product of (anti-)parallel vectors leads to zero-length vectors so you will need to test `angle` for a value of nearly 0.0 or 180.0. If so, `rotationMatrix == identity`.

[The intuition here is the length of the cross-product vector is the area of the implied parallelogram formed by the two vectors.](https://www.geogebra.org/m/psMTGDgc)